.github/workflows/check-shell.yml

```diff
@@ -12,3 +12,14 @@ jobs:
     - run: |
         make validate-shell
       name: Verify Shell scripts
+    - run: |
+        make unit-test-shell
+      name: Unit test Shell scripts
+    - name: Publish Test Results
+      uses: EnricoMi/publish-unit-test-result-action@v2
+      if: always()
+      with:
+        files: |
+          test-results/**/*.xml
```

Comment:
I forgot does the glob pattern `**` also looks in the current dir ? Since `test-results/bats_shell.junit.xml` is not found in any subdirectory